File: src/main/java/no/nav/syfo/controller/domain/sykepengesoknad/AnnenInntektskilde.java
package no.nav.syfo.controller.domain.sykepengesoknad;

import java.io.Serializable;

public class AnnenInntektskilde implements Serializable {
    public boolean sykmeldt;
    public AnnenInntektskildeType annenInntektskildeType;

    public AnnenInntektskilde withSykmeldt(final boolean sykmeldt) {
        this.sykmeldt = sykmeldt;
        return this;
    }

    public AnnenInntektskilde withAnnenInntektskildeType(final AnnenInntektskildeType annenInntektskildeType) {
        this.annenInntektskildeType = annenInntektskildeType;
        return this;
    }
}

File: src/main/kotlin/no/nav/syfo/controller/domain/sykepengesoknad/AnnenInntektskilde.kt
package no.nav.syfo.controller.domain.sykepengesoknad

import java.io.Serializable

class AnnenInntektskilde : Serializable {
    var sykmeldt = false
    var annenInntektskildeType: AnnenInntektskildeType? = null
    fun withSykmeldt(sykmeldt: Boolean): AnnenInntektskilde {
        this.sykmeldt = sykmeldt
        return this
    }

    fun withAnnenInntektskildeType(annenInntektskildeType: AnnenInntektskildeType?): AnnenInntektskilde {
        this.annenInntektskildeType = annenInntektskildeType
        return this
    }
}
